src/styles/default.css

```
@@ -8,15 +8,28 @@
   Touched: 2021
 */
 
-.hljs {
+/*
+This is left on purpose making default.css the single file that can be lifted
+as-is from the repository directly without the need for a build step
+
+Typically this "required" baseline CSS is added by `makestuff.js` during build.
+*/
+pre code.hljs {
   display: block;
   overflow-x: auto;
   padding: 1em;
+}
+
+code.hljs {
+  padding: 3px 5px;
```

People use all sorts of different high-res (or low-res) displays these days; should this be `pt`-based instead of `px`-based?

```suggestion
  padding: 4pt 6pt;
```

---

Curious, what cases/devices are you thinking of here?  Everything I'm aware of accounts for Retina (2x, 3x, etc) screens automagically so the 3px will already be 6px or 9px or whatever is needed to derive the correct 3px-ish appearance regardless of screen pixel density.